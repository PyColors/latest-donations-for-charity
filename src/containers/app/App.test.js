import React from 'react';
test('Render a main', () => {
    const wrapper = shallow(
        <main>Hello Jest!</main>
    );
    expect(wrapper).toMatchSnapshot();
});
