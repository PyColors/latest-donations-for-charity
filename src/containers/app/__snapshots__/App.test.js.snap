// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a main 1`] = `
<main>
  Hello Jest!
</main>
`;
